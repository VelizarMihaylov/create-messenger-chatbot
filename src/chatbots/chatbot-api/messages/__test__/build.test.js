import buildMessages from '../build'

describe('buildMessages', () => {
  it('should build message array with id', () => {
    const messages = [
      id => ({
        id,
        text: 'First message'
      }),
      id => ({
        id,
        text: 'Second message'
      })
    ]
    expect(buildMessages(messages)('23461')).toMatchSnapshot()
  })
})
