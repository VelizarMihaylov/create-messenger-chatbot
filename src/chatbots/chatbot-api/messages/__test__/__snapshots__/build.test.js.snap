// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`buildMessages should build message array with id 1`] = `
Array [
  Object {
    "id": "23461",
    "text": "First message",
  },
  Object {
    "id": "23461",
    "text": "Second message",
  },
]
`;
